.github/workflows/ci.yaml

```diff
@@ -142,3 +141,24 @@ jobs:
           --exclude rosetta-server-ethereum \
           --exclude rosetta-server-polkadot \
           --exclude rosetta-client
+
+    - name: Cleanup Docker
+      run: ./scripts/reset_docker.sh
+  
+  #Running arbitrum nitro-testnode
+  arbitrum-nitro-testnode-script:
```

Comment:
Github assumes that each task may run in a different machine, so is not advised to use one task as setup of another task, instead you must run the tests here instead just do the setup.

Reply:
Arbitrum separate step removed, and added into the tests.